woocommerce/woocommerce-gutenberg-products-block
# Cart & Checkout: Update styles to better match designs

Fixes #1539.

This PR:
* Updates styles to better match designs. I reviewed font size, font weight, margins, alignments, sizes, borders, etc. There might still be errors, but the layout should be closer to the designs than it was.
  * I changed some font colors, but didn't spend much time on it because font color will be handled in #1964.
* I converted several font-sizes, line-heights, etc. to relative sizes for a11y (using rem instead of hardcoded px).
* There are some minor JS fixes here and there.

### Screenshots

_Cart (before):_
![imatge](https://user-images.githubusercontent.com/3616980/80121686-ead58980-858c-11ea-9689-acdb86c81b35.png)

_Cart (after):_
![imatge](https://user-images.githubusercontent.com/3616980/80121410-8f0b0080-858c-11ea-83ed-c85743403035.png)

_Checkout (before):_
![imatge](https://user-images.githubusercontent.com/3616980/80121850-12c4ed00-858d-11ea-9ab8-566743026bfe.png)

_Checkout (after):_
![imatge](https://user-images.githubusercontent.com/3616980/80121583-c8dc0700-858c-11ea-82d5-0570f75d4d50.png)

### How to test the changes in this Pull Request:

1. Open the _Cart_ and _Checkout_ blocks with any browser and verify the layout is like in the designs.
2. Bonus points if you test mobile and desktop.
3. Bonus points if you change your browser default font size (layout will probably break a bit, I think that's unavoidable, but all fonts should scale properly).
4. Bonus points if you create a product with a very long word and verify the word breaks into several lines instead of overflowing:
<img src="https://user-images.githubusercontent.com/3616980/80121210-4a7f6500-858c-11ea-8b11-109a6e60b985.png" width="321" />
